index.html

```diff
@@ -1155,20 +1155,20 @@ <h2>Constrainable Properties</h2>
       </table>
     </section>
     <section class="informative">
-      <h2>Timestamp concepts</h2>
+      <h2>Video timestamp concepts</h2>
       <p>
-        Media flowing inside media stream tracks comprise of a sequence of media frames, where
+        Video media flowing inside media stream tracks comprise of a sequence of video frames, where
         the frames are sampled from the media at instants spread out over time.
       </p>
+      <p>
+        Each video frame has a <dfn class="export">presentation timestamp</dfn>
+        which is relative to the first frame appearing on the track.
+      </p>
       <p>
         Each frame may have a <dfn class="export">capture timestamp</dfn> representing
         the instant it was captured. A source of frames can define how this timestamp is set, otherwise
         it is unset.
       </p>
-      <p>
-        Each video frame has a <dfn class="export">presentation timestamp</dfn>
-        which is relative to the first frame appearing on the track.
-      </p>
       <p>
         Each frame may have a <dfn class="export">receive timestamp</dfn> representing
         the last received timestamp of packets used to produce this video frame was received in its entirety.
```

Comment:
Since the rest of the text seems to assume a locally captured MST, introducing receiveTime raises questions.  If you are going to also cover remote tracks, then there will need to be text talking about which Wallclock is being used, whether there is an adjustment from the sender to receiver clock, etc.

Reply:
This piece of the text is defining unobservable "internal slots" and can't hold specifics or behavior; the thought is to do that for each source and sink instead - based on the TPAC session I thought there was consensus for this way?

Regarding wallclock, this is defined for the concrete VideoFrameMetadata below this section.